client/lib/plans/features-list.js

```diff
@@ -801,52 +801,52 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_CORE_JETPACK ]: {
 		getSlug: () => constants.FEATURE_CORE_JETPACK,
-		getTitle: () => i18n.translate( 'Core Jetpack Services' ),
+		getTitle: () => i18n.translate( 'Core Jetpack services' ),
 		getDescription: () => i18n.translate( 'Stats, themes, and promotion tools.' ),
 		hideInfoPopover: true,
 	},
 	[ constants.FEATURE_BASIC_SECURITY_JETPACK ]: {
 		getSlug: () => constants.FEATURE_BASIC_SECURITY_JETPACK,
-		getTitle: () => i18n.translate( 'Basic Security' ),
+		getTitle: () => i18n.translate( 'Basic security' ),
 		getDescription: () =>
 			i18n.translate( 'Brute force protection, monitoring, secure logins, updates.' ),
 		hideInfoPopover: true,
 	},
 
 	[ constants.FEATURE_BASIC_SUPPORT_JETPACK ]: {
 		getSlug: () => constants.FEATURE_BASIC_SUPPORT_JETPACK,
-		getTitle: () => i18n.translate( 'Basic Support' ),
+		getTitle: () => i18n.translate( 'Basic support' ),
 		getDescription: () => i18n.translate( 'Free support to help you make the most of Jetpack.' ),
 		hideInfoPopover: true,
 	},
 
 	[ constants.FEATURE_SPEED_JETPACK ]: {
 		getSlug: () => constants.FEATURE_SPEED_JETPACK,
-		getTitle: () => i18n.translate( 'Speed and Storage' ),
+		getTitle: () => i18n.translate( 'Speed and storage' ),
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.